package/Shaders/Lighting.hlsl

```diff
@@ -1646,13 +1666,13 @@ PS_OUTPUT main(PS_INPUT input, bool frontFace
 
 	wetnessNormal = normalize(lerp(wetnessNormal, float3(0, 0, 1), saturate(flatnessAmount)));
 
+	float3 rippleNormal = normalize(lerp(float3(0, 0, 1), raindropInfo.xyz, clamp(flatnessAmount, .2, 1)));
+	wetnessNormal = ReorientNormal(rippleNormal, wetnessNormal);
+
 	float waterRoughnessSpecular = 1.0 - wetnessGlossinessSpecular;
 
-#		if defined(WETNESS_EFFECTS)
```

Comment:
Is this intended? What happens if wetness_effects isn't defined?

Reply:
This is an `if defined(WETNESS_EFFECTS)` inside another `if defined(WETNESS_EFFECTS)` i.e. redundant